
<launch>
    <node pkg="lidar_accumulator" type="lidar_accumulator_2" name="lidar_accumulator_2" output="screen">
        <param name="s_input_point_topic_name_" value="/velodyne_points"/>
        <!-- <param name="s_input_point_topic_name_" value="/afi910_cloud_node/cloud"/> -->
        <param name="s_input_odom_topic_name_" value="/odometry_node/odometry"/>
        <param name="s_output_point_topic_name_" value="accum_points"/>

        <param name="cfg_i_accum_num_lidar_scan_" value="30"/>

        <param name="cfg_d_accum_key_threshold_m_" value="0.4"/>

        <param name="cfg_d_roi_min_m_" value="5.0"/>
        <param name="cfg_d_roi_max_m_" value="20.0"/>

        <param name="cfg_b_filter_use_voxel_" value="true"/>
        <param name="cfg_d_filter_voxel_leaf_" value="0.15"/>

        <param name="cfg_b_msg_debug_" value="true"/>
        <param name="cfg_b_msg_comp_time_" value="true"/>
        
    </node>

    <!-- <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node pkg="rviz" type="rviz" name="rviz" args="-d $(find lidar_odometry)/lidar_odom.rviz" />
    </group> -->

</launch>
